EIPS/eip-.md

```diff
@@ -0,0 +1,128 @@
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-08-03
+requires: 211, 1014, 3855
+---
+
+## Abstract
+
+A minimal `CREATE2` factory is inserted at a chosen address to enable deterministic deployments across EVM chains.
+
+## Motivation
+
+There are now a large number of EVM chains where users want to transact and developers want to deploy applications, and we can expect this number to continue to grow in line with Ethereum's rollup-centric roadmap and the general adoption of programmable blockchains.
+
+Most applications support multiple chains and aspire to support as many as possible. Their developers widely prefer to deploy contracts at identical addresses across all chains, which we'll call a *multi-chain deterministic deployment*.
+
+This kind of deployment reduces the number of addresses that must be distributed to use the application, so that it no longer scales with the number of supported chains. This simplification has many benefits throughout the stack: interfaces and SDKs need to embed and trust fewer addresses, and other contracts that depend on them do not require chain-specific customization (which in turn makes them amenable to multi-chain deployment).
+
+Another important motivation is account abstraction. Accounts tied to a single chain are difficult to explain to users and can cause loss of funds. Smart contract accounts must be multi-chain like EOAs or they offer downgraded UX and are more prone to error.
+
+There is currently no native or fully robust way to perform multi-chain deterministic deployments. While `CREATE2` enables deterministic deployments, the created address is computed from that of the contract that invokes the instruction, so a *factory* that is itself multi-chain is required for bootstrapping. Four workarounds are currently known to deploy such a factory, each with their own issues:
+
+1. A keyless transaction is crafted using Nick's method that can be posted permissionlessly to new chains. For this to work, the chain must support legacy transactions without [EIP-155](./eip-155.md) replay protection, and the fixed gas price and gas limit must be sufficiently high, but not so high as to exceed the limits of the chain.
+2. Private keys held by some party are used to sign creation transactions for each chain as needed. This creates a dependency on that party, does not provide a hard guarantee that the factory will be available on every chain, and can also irreversibly fail if transactions are not properly parameterized.
+3. [ERC-7955](./eip-7955.md): A private key is intentionally leaked so that any party can permissionlessly create an [EIP-7702](./eip-7702.md) signed delegation and deploy a factory from the leaked account. While this approach improves on the previous two, its reliance on ECDSA keys makes it non-quantum-resistant, and will fail once chains stop supporting ECDSA keys.
```

Comment:
Pending merge: https://github.com/ethereum/ERCs/pull/1052

Reply:
>which is loosely planed AFAIU

Will add the note but I actually don't think this is planned or likely. Because many contracts rely on ECRECOVER for authorization (eg, Permit), permanently setting a delegation doesn't effectively change the account logic. Messing with ECRECOVER to patch that is extremely unlikely.